src/libs/ajax.js

```diff
@@ -46,6 +47,7 @@ const withUrlPrefix = _.curry((prefix, wrappedFetch) => (path, ...args) => {
 export const fetchOk = _.flow(withInstrumentation, withCancellation, withErrorRejection)(fetch)
 
 const fetchCbas = withUrlPrefix(`${getConfig().cbasUrlRoot}/api/batch/v1/`, fetchOk)
+const fetchCromwell = withUrlPrefix(`${getConfig().cbasUrlRoot}/cromwell/api/workflows/v1/`, fetchOk)
```

Comment:
I have not tested it but I believe that structuring the path this way should make it Just Work in the app.